External/Notepad/Theme.swift

```diff
@@ -96,11 +96,11 @@ private extension Theme {
 private extension Theme {
 
     private static var fontSize: CGFloat {
-        var fontSize = CGFloat(UserDefaults.standard.integer(forKey: "kFontSizePreferencesKey"))
+        var fontSize = CGFloat(Options.shared.fontSize)
```

Comment:
Q: Can we have the `fontSize` be a `CGFloat` directly, rather than converting here?